### DISCUSSION TITLE
Is there away to log in YT so age restriction won't apply?

### DISCUSSION THREAD

**Discussion by arcovoltaico** [fent/node-ytdl-core#1116] [Q&A]
How I can I make node-ytdl-core use my credentials so hopefully I will overcome the age restriction?
I think there was another strategy being tested[ here ](https://github.com/fent/node-ytdl-core/commit/7b26eb59ba818312f3df5f19dfbf9def726c809c) What's the current situation?
 Cheers!

**Comment by WaqasIbrahim** [comment]
You can grab your cookie from youtube when you are logged in using an extension like:
https://chrome.google.com/webstore/detail/get-cookiestxt/bgaddhkoddajcdgocldbbfleckgcbcid?hl=en
And the follow this example:
https://github.com/fent/node-ytdl-core/blob/master/example/cookies.js
